README.org

```diff
@@ -166,6 +166,7 @@ and could act as examples:
 | [[https://github.com/DamienCassou/libelcouch][libelcouch]]     | Library to communicate with [[https://couchdb.apache.org/][CouchDB]] databases                                                     |
 | [[https://github.com/DamienCassou/elcouch][elcouch]]        | User interface to view and manipulate [[https://couchdb.apache.org/][CouchDB]] databases                                           |
 | [[https://github.com/DamienCassou/khardel][khardel]]        | User interface to integrate [[https://github.com/scheibler/khard][khard]], a console cardav client                                        |
+| [[https://github.com/tarides/ocaml-eglo][ocaml-eglot]]    | Minor mode for editing OCaml using LSP through Eglot                                              |
```

Comment:
there is a typo in the URL

Reply:
Ah yes! (I always have trouble writing links with ORG-mode, because I write `[[past]` and paf the link is inlined).

Fixed on the last push!

Reply:
I always use `C-c C-l` to create and edit Org links